extensions/src/ai_storybook_dancing/index.ts

```diff
@@ -30,7 +30,7 @@ export default class ExtensionNameGoesHere extends extension(details, "blockly",
   }
 
   protected override saveDataHandler = new SaveDataHandler({
```

Comment:
NOTE to @jimbo797 : This can be deleted once we have finalized the project we want to load with the activity. I made sure to capture this idea here: https://mitjibo.atlassian.net/browse/AS-11